src/main/java/mocacong/server/domain/Comment.java

```diff
@@ -62,4 +70,22 @@ public void updateComment(String content) {
     public void removeMember() {
         this.member = null;
     }
+
+    public void incrementCommentReport(Member reporter, String reportReason) {
+        if (!hasAlreadyReported(reporter)) {
+            ReportReason reason = ReportReason.from(reportReason);
+            this.reports.add(new CommentReport(this, reporter, reason));
+        } else {
+            throw new DuplicateReportCommentException();
+        }
+    }
```

Comment:
else문은 제거 가능할 것 같아요.
```suggestion
    public void incrementCommentReport(Member reporter, String reportReason) {
        if (!hasAlreadyReported(reporter)) {
            ReportReason reason = ReportReason.from(reportReason);
            this.reports.add(new CommentReport(this, reporter, reason));
        }
        throw new DuplicateReportCommentException();
    }
```